docs/app/Examples/elements/Icon/IconSet/IconCategoryExample.js

```diff
@@ -16,12 +16,6 @@ const IconCategoryExample = ({ category }) => (
 )
 
 IconCategoryExample.propTypes = {
-  /** An element type to render as (string or function). */
-  as: PropTypes.oneOfType([
-    PropTypes.string,
-    PropTypes.func,
-  ]),
-
```

Comment:
The `as` prop was added to all components via regex.  Some received the as prop where it didn't make sense, like in doc components.  This PR also removes those.